contrib/hamilton/contrib/user/zilto/llm_generate_code/__init__.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+import subprocess
+from typing import Optional
+
+from hamilton.function_modifiers import extract_fields
+
+logger = logging.getLogger(__name__)
+
+from hamilton import contrib
+
+with contrib.catch_import_errors(__name__, __file__, logger):
+    import openai
+
+
+def llm_client(api_key: Optional[str] = None) -> openai.OpenAI:
+    """Create an OpenAI client"""
+    if api_key is None:
+        api_key = os.environ.get("OPENAI_API_KEY")
+
+    return openai.OpenAI(api_key=api_key)
+
+
+def prompt_template_to_generate_code() -> str:
+    return """Write some {code_language} code to solve the user's problem.
+
+Return only python code in Markdown format, e.g.:
+
+```{code_language}
+....
+```
+
+user problem
+{query}
+
+{code_language} code
+"""
```

Comment:
have a look at how I'm doing multi line - let me know what you think.
My issue with this is that it makes things look uglier IMO.

Reply:
For continuous text, I think I still prefer my approach with `"""`. But I will switch to how you do with for Python code (in `code_prepared_for_execution` since Python statements need to be on separate lines